itest/rust/src/object_tests/call_deferred_test.rs

```diff
@@ -34,6 +38,9 @@ impl DeferredTestNode {
     fn accept_gd(mut this: Gd<Self>) {
         this.set_name(ACCEPTED_NAME);
     }
+    fn accept_dyn_gd(mut this: DynGd<Object, dyn ForwardTrait>) {
+        this.dyn_bind_mut().forward();
+    }
```

Comment:
Please apply suggestions directly, this added an empty line before that is now missing again 🙂